osgp/platform/osgp-adapter-domain-core/src/main/java/org/opensmartgridplatform/adapter/domain/core/application/services/FirmwareManagementService.java

```diff
@@ -86,6 +86,7 @@
 
     @Value("${ssld.pending.firmware.update.get.firmware.version.delay}")
     private Long getFirmwareVersionDelay;
+    private SsldPendingFirmwareUpdate ssldPendingFirmwareUpdate;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "ssldPendingFirmwareUpdate" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1068)